projects/odf_transform/odf_transform/config.json

```diff
@@ -1,6 +1,6 @@
 {
-    "odf_path":"../sample_data/test_files/",
-    "output_path":"./output/",
+    "odf_path":"D:\\review_ODF",
```

Comment:
would be best to keep this in the relative path & unix format so we can do automated testing

Reply:
oh yeah that's note suppose to be there

Reply:
This is now back to the original test path